services/ui-src/src/components/banners/AdminBannerProvider.tsx

```diff
@@ -1,28 +1,35 @@
 import { useState, createContext, ReactNode, useMemo, useEffect } from "react";
 // utils
 import { AdminBannerData, AdminBannerShape } from "utils/types/types";
-import { bannerId } from "../../utils/constants/constants";
+import { bannerId, GET_BANNER_FAILED } from "../../utils/constants/constants";
+import { errorHandler } from "utils/errors/errorHandler";
 // api
 import { deleteBanner, getBanner, writeBanner } from "utils/api/index";
 
 const ADMIN_BANNER_ID = bannerId;
 
 export const AdminBannerContext = createContext<AdminBannerShape>({
   bannerData: {} as AdminBannerData,
-  fetchAdminBanner: () => {},
-  writeAdminBanner: () => {},
-  deleteAdminBanner: () => {},
+  fetchAdminBanner: Function,
+  writeAdminBanner: Function,
+  deleteAdminBanner: Function,
+  errorData: null,
 });
 
 export const AdminBannerProvider = ({ children }: Props) => {
   const [bannerData, setBannerData] = useState<AdminBannerData>(
     {} as AdminBannerData
   );
+  const [errorState, setErrorState] = useState(null);
 
   const fetchAdminBanner = async () => {
-    const currentBanner = await getBanner(ADMIN_BANNER_ID);
-    const newBannerData = currentBanner?.Item || {};
-    setBannerData(newBannerData);
+    try {
+      const currentBanner = await getBanner(ADMIN_BANNER_ID);
+      const newBannerData = currentBanner?.Item || {};
+      setBannerData(newBannerData);
+    } catch (error) {
+      errorHandler(error, setErrorState, GET_BANNER_FAILED);
+    }
```

Comment:
You'll see this kind of pattern a lot. Added a try/catch with the `errorHandler` doing it's thing.